src/goto-symex/symex_function_call.cpp

```diff
@@ -125,7 +125,21 @@ void goto_symext::parameter_assignments(
         }
       }
 
-      symex_assign(state, code_assignt(lhs, rhs));
+      assignment_typet assignment_type;
```

Comment:
In #3459 it says that this PR might be a candidate to test this, but there isn't a test here either?

Reply:
Now there! (Note absence of assignments to parameters in the trace).

Reply:
The test just seems to test what should be there, but does not have any pattern describing strings that should _not_ occur (other than the usual "warning: ignoring")?!

Reply:
Ok, will revert to the state without #3459, and will then add the test there (after this one here is merged). The diff is then obvious.